src/components/registration/Checkbox.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment, useContext } from 'react';
+import { FormGroup, Form } from 'react-bootstrap';
+import RegistrationContext from '../../contexts/registration/RegistrationContext';
+
+const Checkbox = () => {
+  const registrationContext = useContext(RegistrationContext);
+  const { termsAccepted, setTermsAccepted } = registrationContext;
```

Comment:
To można spokojnie połączyć - 
```suggestion
  const { termsAccepted, setTermsAccepted } = useContext(RegistrationContext);
```